lib/tasks/getTemplates.js

```diff
@@ -15,15 +15,15 @@ module.exports = function getTemplates(folderPath) {
   var walked = walkSync(path.join(folderPath, 'templates'));
 
   //Only leave files (plus their relative path) in the walked array
-  var walkedFiles = walked.filter(function(element){
+  var walkedFiles = walked.filter(function (element){
```

Comment:
instead of `function (xx){` I would do it `function(xx) {`
space before the opening curly.